47.md

```diff
@@ -126,6 +127,224 @@ Response:
 Errors:
 - `PAYMENT_FAILED`: The payment failed. This may be due to a timeout, exhausting all routes, insufficient capacity or similar.
 
+### `multi_pay_invoice`
+
+Description: Requests payment of multiple invoices.
+
+Request:
+```jsonc
+{
+    "method": "multi_pay_invoice",
+    "params": {
+        "invoices": [
+          "lnbc50n1..." // bolt11 invoice
+        ],
+    }
+}
+```
+
+Response:
+
+For every invoice in the request, a separate response event is sent. To differentiate between the responses, each
+response event contains an `payment_hash` tag with the payment hash of the invoice it is responding to.
+
+```jsonc
+{
+    "result_type": "multi_pay_invoice",
+    "result": {
+        "preimage": "0123456789abcdef..." // preimage of the payment
+    }
+}
+```
+
+Errors:
+- `PAYMENT_FAILED`: The payment failed. This may be due to a timeout, exhausting all routes, insufficient capacity or similar.
+
+### `pay_keysend`
+
+Request:
+```jsonc
+{
+    "method": "pay_keysend",
+    "params": {
+        "amount": 123, // invoice amount in msats, required
+        "pubkey": "03...", // payee pubkey, required
+        "preimage": "0123456789abcdef...", // preimage of the payment, optional
+        "tlv_records: [ // tlv records, optional
+            {
+                "type": 5482373484, // tlv type
+                "value": "fajsn341414fq" // hex encoded tlv value
+            }
+        ]
+    }
+}
+```
+
+Response:
+```jsonc
+{
+    "result_type": "pay_keysend",
+    "result": {
+        "preimage": "0123456789abcdef...", // preimage of the payment
+    }
+}
+```
+
+### `make_invoice`
+
+Request:
+```jsonc
+{
+    "method": "make_invoice",
```

Comment:
@benthecarman regarding
> Yeah I guess you might not anyone with a NWC connection to be able to create with custom preimages

Are we able to remove these fields from the `make_invoice` request for now?
- `invoice`
- `preimage`
- `payment_hash`

Reply:
Update: are some of these fields incorrectly taken from the response?

Reply:
```json
"params": {
  "amount": 123, // value in msats
   "description": "string", // invoice's description, optional
   "description_hash": "string", // invoice's description hash, optional
   "expiry": 213 // expiry in seconds from time invoice is created
}
```

Reply:
Fixed